oauth2-server/src/main/java/com/clouway/oauth2/IssueNewTokenActivity.java

```diff
@@ -57,14 +57,21 @@ public Response execute(Client client, Identity identity, Set<String> scopes, Re
       Pem.Block key = certificates.get(certKey);
       String host = request.header("Host");
 
+      Map<String, Object> claims = new LinkedHashMap<>();
+      claims.put("iss", host);
+      claims.put("aud", client.id);
+      claims.put("sub", identity.id());
+      claims.put("iat", instant.asDate());
+      claims.put("exp", new Date(accessToken.expirationTimestamp()));
+      claims.put("name", identity.name());
+      claims.put("email", identity.email());
+      claims.put("given_name", identity.givenName());
+      claims.put("family_name", identity.familyName());
+      claims.putAll(identity.claims());
```

Comment:
Have you checked that claims.putAll works with all primitive types such as boolean, long, integer, string  ?

There was a check with GSON but it need to be verified and here.